docs/workforce-management-framework/topics/con-wfm-extending-user-interface.adoc

```diff
@@ -1,18 +1,18 @@
 [id='{context}-con-extending-the-user-interface']
 = Extending the User Interface
 
-{PROJECT_NAME} framework provides a set of extension points which allows a developer to modify an Angular User Interface (UI).
-All modifications are done by supplying additional modules which provide a set of the Angular.js directives and are used to extend the Mobile and Portal UI.
+{PROJECT_NAME} framework provides a set of extension points that allows a developer to modify an Angular User Interface (UI).
+You can modify the UI by writing additional modules that provide a set of the Angular.js directives to extend the Mobile and Portal UI.
 The {PROJECT_NAME} Application is shipped with the `angularjs-extensions` example that extends the Mobile and Portal Applications:
 
  * https://github.com/feedhenry-raincatcher/raincatcher-angularjs/tree/master/packages/angularjs-extensions
 
 This Module contains the example directives that are used for demo purposes and it can be used as a template to extend the existing {PROJECT_NAME} UI.
 
-== Extending Workorders
+== Extending the Workorders UI
 
 By default, {PROJECT_NAME} provides a minimal set of defined fields for a _workorder_.
-Depending on the needs of the business, a developer can add additional data and UI elements, for example, gps location, an address or a description UI element.
+If your step results contain additional data, for example, gps location, an address or a description UI element, you probably want to extend the workorders UI to display that data.
```

Comment:
If your step results contain additional data, for example, gps location, an address or a description UI element, you can extend the workorder's UI to display this data.